bescort.lic

```diff
@@ -210,6 +213,8 @@ class Bescort
       take_xing_ferry(args.mode)
     elsif args.ferry1
       take_ain_ghazal_ferry(args.mode)
+	  elsif args.lang_barge                     
```

Comment:
It looks like you've got some tabs in here, could you adjust your editor to use spaces instead?

Another option would be to run Rubocop: `rubocop -a`

Reply:
Wow you're fast, sorry I'll update to spaces